src/main/java/fret/Fret.java

```diff
@@ -0,0 +1,59 @@
+package fret;
+
+import command.Command;
+import javafx.util.Pair;
+import psu.Storage;
+import psu.Ui;
+
+import task.TaskList;
+
+public class Fret {
+    Storage storage;
+    Ui fretUi;
+    TaskList userTasks;
+
+    public Fret(String filepath) {
+        this.storage = new Storage(filepath);
+        this.fretUi = new Ui();
+        this.userTasks = new TaskList(storage.loadTasksFromMemory());
+    }
+
+    public String welcomeUser() {
+        return fretUi.welcomeUser();
+    }
+
+    /**
+     * Runs the Fret chatbot program
```

Comment:
Should there be a period at the end of Javadoc lines?

Reply:
Oh yes you are right. Thanks for pointing it out!